.travis.yml

```diff
@@ -2,7 +2,6 @@ language: dart
 sudo: false
 dart:
   - "dev/raw/latest"
-  - stable
```

Comment:
We may not need to omit this; I think at least the analyzer would be happy with both 2.01 and 2.1, but your call here.

Reply:
Restored testing with stable, as it seems to work.